src/api/retrieveQuizQuestions.js

```diff
@@ -0,0 +1,91 @@
+//const axios = require('axios').default;
+
+const questionsApis = {
+  DOGS: { url: 'https://api.thedogapi.com/v1/breeds', api_key: '62b8cc9a-2c13-4eec-abe2-80703eabb0a6' },
+  CATS: { url: 'https://api.thecatapi.com/v1/breeds', api_key: '2d4cf1ee-1883-474f-80ab-f931262fd79b' },
```

Comment:
API keys should be stored in environmental variables. You should move it to the .env file (it must be ignored by git).

...but they are already exposed, so we can consider them compromised.